docs/index.rst

```diff
@@ -114,6 +114,14 @@
 .. toctree::
   :maxdepth: 0
   :hidden:
+  :caption: Stackdriver Error Reporting
```

Comment:
Is `Stackdriver` part of the public branding?  We don't use it for logging or monitoring, for instance.